docs/algorithms/sig/falcon.yml

```diff
@@ -17,7 +18,7 @@ website: https://falcon-sign.info
 nist-round: 3
 spec-version: v1.2
```

Comment:
KATs changed -- so would a spec version update not be called for?

Reply:
The spec of Falcon is a complicated story. The Falcon 1.2 PDF specifies different lengths of signatures than what the implementation has used. This PR switches over to the shorter signature lengths, so actually we're bringing it in spec with 1.2?

On the other hand, there's no way we're compatible with the NIST KATs now, because those still use the 690-byte signatures.

Reply:
Is there any way to get input from the authors of this algorithm? How do they refer to the different algorithm versions, if not by version ID? If this implementation is not in line with the NIST KATs, does reference to NIST make any sense for this code version? How is this code base then to be viewed relative to the NIST competition? "between rounds"? "unrelated"?

Reply:
I don't think they're being that consistent about it. The implementation simply supports a bunch of different encodings of signatures. I'm not sure why the signatures were still set at the longer lengths, in the NIST submission, but I don't think it's necessary anymore.

Reply:
So, how do we conclude this then?